p2p/connmgr/banscore.go

```diff
@@ -0,0 +1,22 @@
+package connmgr
+
+// Score of penalty node
+const (
```

Comment:
Please add a new type such as `BanScore` for the new xxxScore `const` definitions. Please avoid using the raw `int` type to guarantee the compile-time type safety.

Reply:
https://github.com/Qitmeer/qitmeer/pull/255/commits/b8470458f82c4b88dfa42aa2e84fa3b7026b8f20